src/devices/Vl53L1X/Vl53L1X.cs

```diff
@@ -0,0 +1,959 @@
+                    .VL53L1X_RESULT__PEAK_SIGNAL_COUNT_RATE_CROSSTALK_CORRECTED_MCPS_SD0) * 8);
+            }
+        }
+
+        /// <summary>
+        /// Returns the current number of enabled SPADs.
+        /// </summary>
+        public ushort SpadNb
+        {
+            get
+            {
+                return (ushort)(ReadUInt16((ushort)Registers.VL53L1X_RESULT__DSS_ACTUAL_EFFECTIVE_SPADS_SD0) >> 8);
+            }
+        }
+
+        /// <summary>
+        /// Returns the ambient rate in kcps.
+        /// </summary>
+        public ushort AmbientRate
+        {
+            get
+            {
+                return (ushort)(ReadUInt16((ushort)Registers.RESULT__AMBIENT_COUNT_RATE_MCPS_SD) * 8);
+            }
+        }
+
+        /// <summary>
+        /// Returns the <see cref="RangeStatus" /> of the device.
+        /// </summary>
+        public RangeStatus RangeStatus
+        {
+            get
+            {
+                byte rgSt = ReadByte((ushort)Registers.VL53L1X_RESULT__RANGE_STATUS);
+
+                rgSt = (byte)(rgSt & 0x1F);
+
+                switch (rgSt)
+                {
+                    case 9:
+                        return RangeStatus.NoError;
+                    case 6:
+                        return RangeStatus.SigmaFailure;
+                    case 4:
+                        return RangeStatus.SignalFailure;
+                    case 5:
+                        return RangeStatus.OutOfBounds;
+                    case 7:
+                        return RangeStatus.WrapAround;
+                    default:
+                        throw new Exception($"The returned range status code {rgSt} of the device is unknown.");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the offset correction value in mm.
+        /// </summary>
+        public short Offset
+        {
+            get
+            {
+                short offset = ReadInt16((ushort)Registers.ALGO__PART_TO_PART_RANGE_OFFSET_MM);
+                offset <<= 3;
+                offset /= 32;
+                return offset;
+            }
+            set
+            {
+                WriteInt16((ushort)Registers.ALGO__PART_TO_PART_RANGE_OFFSET_MM, (short)(value * 4));
+                WriteInt16((ushort)Registers.MM_CONFIG__INNER_OFFSET_MM, 0x0);
+                WriteInt16((ushort)Registers.MM_CONFIG__OUTER_OFFSET_MM, 0x0);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the crosstalk correction value in cps.
+        /// This is the number of photons reflected back from the cover glass in cps.
+        /// </summary>
+        public ushort Xtalk
+        {
+            get
+            {
+                ushort xTalk = ReadUInt16((ushort)Registers.ALGO__CROSSTALK_COMPENSATION_PLANE_OFFSET_KCPS);
+
+                return (ushort)((xTalk * 1000) >> 9); /* * 1000 to convert kcps to cps and >> 9 (7.9 format) */
+            }
+            set
```

Comment:
nit, you need an extra CR between } and set.

Reply:
Done